src/app/contact-page/contact-page.component.css

```diff
@@ -0,0 +1,17 @@
+#form-container {
+  margin: 0 auto;
+  width: 60%;
+  background-color: #fff;
+  padding: 40px;
+  margin-top: 30px;
+}
+
+#button-submit-ticket {
+  border-radius: 0%;
+  float: right;
```

Comment:
All of the other buttons are aligned left. We should be consistent and do the same here.